Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,61 @@
 ﻿using System;
+using static System.Math;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+                /*  Homework #1 
+                    Read name, surename, age, weight (in kg) and height (in cm) from console.
+                    1) Print all the info based on the example message below:
+                        Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm. 
+                    2) Calculate and print body-mass index (BMI)
+                    3) Do 1 and 2 for another person.
+                */
+
+        private static string name, surname, decision = "Y";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'surname' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832242)